### PR TITLE
i18n(zh-cn): Update `configuration-reference.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

Update `configuration-reference.mdx`

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9240

### CODE REVIEW

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -156,15 +156,14 @@ export default defineConfig({
...
+**类型**：`'static' | 'server'`<br />
 **默认值**：`'static'`
 </p>
 
 指定构建的输出目标。
 
-- `static`- 构建静态网站，部署到任何静态托管服务器上。
-- `server` - 构建应用，部署到支持 SSR（服务器端渲染）的托管服务器上。
-- `hybrid` - 构建包含少量服务端渲染页面的静态网站。
+- `static`- 默认预渲染你的所有页面，如果没有页面选择退出预渲染的话，将输出一个完全静态的站点。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:165]:
```suggestion
- `static` - 默认预渲染你的所有页面，如果没有页面选择退出预渲染的话，将输出一个完全静态的站点。
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 2]
```diff
@@ -475,17 +476,17 @@ export default defineConfig({
 <p>
 
 **类型**：`string`<br />
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:478]:
```suggestion
**类型：**`string`<br />
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 3]
```diff
@@ -475,17 +476,17 @@ export default defineConfig({
...
-**默认值**：`'./dist/client'`
+**默认值**：`'./client'`
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:479]:
```suggestion
**默认值：**`'./client'`
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 4]
```diff
@@ -498,7 +499,7 @@ export default defineConfig({
 <p>
 
 **类型**：`string`<br />
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:501]:
```suggestion
**类型：**`string`<br />
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx [hunk 5]
```diff
@@ -498,7 +499,7 @@ export default defineConfig({
...
-**默认值**：`'./dist/server'`
+**默认值**：`'./server'`
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:502]:
```suggestion
**默认值：**`'./server'`
```

File: src/content/docs/zh-cn/reference/configuration-reference.mdx
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/configuration-reference.mdx:1]: - [第796行](https://github.com/withastro/docs/pull/10401/files#diff-f004fc3d5617d91ddb493d5066491dccbdd7bce550f7610468ee8be511dfd78eR796)的 `<ViewTransitions />` 也被替换成 `<ClientRouter />` 了。
- [第1091行](https://github.com/withastro/docs/pull/10401/files#diff-f004fc3d5617d91ddb493d5066491dccbdd7bce550f7610468ee8be511dfd78eR1091)有一个缩进的[修改](https://github.com/withastro/docs/pull/9240/files#diff-6b0ed61bf17daf5f370edd2d5a46be46cb8ef5321ac7537a17e71236f0c9028bR1101)。

**Review by Nin3lee — CHANGES_REQUESTED** [review]

File: src/content/docs/zh-cn/reference/configuration-reference.mdx
Comment (liruifengv) [reply src/content/docs/zh-cn/reference/configuration-reference.mdx:1]: 那个缩进我觉得不用修改

**Review by Nin3lee — APPROVED** [review]
LGTM! Amazing work!

**Pull request merged**